modular_skyrat/master_files/code/modules/clothing/under/costume.dm

```diff
@@ -46,6 +46,14 @@
 	greyscale_colors = "#434d7a" //THATS RIGHT, FUCK YOU! THE BATHROBE CAN BE RECOLORED!
 	flags_1 = IS_PLAYER_COLORABLE_1
 
+/obj/item/clothing/suit/costume/skyrat/vic_dresscoat
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	name = "victorian dresscoat"
+	desc = "An elaborate coat to go over an old-Earth Victorian Period dress. Much thinner fabric than you'd expected."
+	icon_state = "vickyred"
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+
```

Comment:
If the base coat isn't available at all in-game, it's best you just make your donator coat a direct subtype of /costume and not bother with this part. Have the icons in the donator .dmis, and the links to those .dmis in your coat's item instead

Reply:
faaair